java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -10,81 +10,81 @@
 import java.util.UUID;
 
 public class TransactionTestUtilities {
-    private static final String key1 = "{key}" + UUID.randomUUID();
-    private static final String key2 = "{key}" + UUID.randomUUID();
-    private static final String key3 = "{key}" + UUID.randomUUID();
-    private static final String key4 = "{key}" + UUID.randomUUID();
-    private static final String key5 = "{key}" + UUID.randomUUID();
-    private static final String key6 = "{key}" + UUID.randomUUID();
-    private static final String key7 = "{key}" + UUID.randomUUID();
-    private static final String key8 = "{key}" + UUID.randomUUID();
-    private static final String value1 = UUID.randomUUID().toString();
-    private static final String value2 = UUID.randomUUID().toString();
-    private static final String value3 = UUID.randomUUID().toString();
-    private static final String field1 = UUID.randomUUID().toString();
-    private static final String field2 = UUID.randomUUID().toString();
-    private static final String field3 = UUID.randomUUID().toString();
+    private static final String stringKey1 = "{test}:stringKey1-" + UUID.randomUUID();
```

Comment:
I suggest only renaming the variable.  I don't see the benefit of renaming the key.

Reply:
It helped for post-mortem debugging the test - I kept the cluster running for querying them.